app/forms/providers/proceeding_interrupts/related_orders_form.rb

```diff
@@ -0,0 +1,63 @@
+module Providers
+  module ProceedingInterrupts
+    class RelatedOrdersForm < BaseForm
+      form_for Proceeding
+
+      ORDER_TYPES = %i[
+        care
+        supervision
+        secure_accommodation
+        placement
+        adoption
+        emergency_protection
+        child_assessment
+        parenting
+        child_safety
```

Comment:
BTW, what was the outcome of the conversation with design about asking whether this "Placement order" proceeding relates to a "placement order" issue?

Reply:
It doesn't! There was a step missing from the Figma... only certain PLF proceedings hit this check page, placement orders (as an example) wouldn't! :smilr